Metagenomics/Remove_human_reads_from_raw_data/Workflow_Documentation/NF_MGRemoveHumanReads-A/workflow_code/Remove_Human_reads.config

```diff
@@ -0,0 +1,40 @@
+//Variables to set:
+
+params.DL_kraken = false  //whether or not to download the human reads database as the first step
+
+params.single_end = false  // single-end reads (false if paired-end)
+
+params.specify_reads = true  //if true, only process reads specified by the sample_id_list
+
+params.sample_id_list = "/workspace/GeneLab_Data_Processing/rmv/unique_sample_ids.txt"  //list of sample IDs to proccess if specify_reads is true
+
+params.reads_dir = "$projectDir/example-reads_PE/"  //directory to find sample reads
+
+params.PE_reads_suffix = "_R{1,2}.fastq.gz"  //raw read suffixes (region following the unique part of the sample names)
+ 						//e.g. for "Sample-1_R1/2_raw.fastq.gz" would be "_R1_raw.fastq.gz"
+
+params.PE_reads_out_suffix = "_R#_raw_hrRemoved.fastq"  //suffix to use for final (human reads removed) output files
+
+
+params.SE_reads_suffix = "_raw.fastq.gz"  //if single-end, set this. raw read suffixes which follow the unique part of sample name
+
+params.SE_reads_out_suffix = "_raw_hrRemoved.fastq"  //suffix to use for final (human reads removed) output files
+
+
+
+
+//Only change if desired:
+
+params.num_threads = 2
+params.kraken_output_dir = "$projectDir/kraken2-outputs"  //location to output files, relative to wd or full path
+params.human_db_name = 'kraken2-human-db'  //
+params.human_db_path = "$projectDir/${params.human_db_name}"

```

Comment:
Again, the $projectDir isn't discussed in the README. In this case, it is also a good idea to make it clear to the user that the db path doesn't necessarily have to be the same as the read path and using "projectDir" for both is confusing, unless they're downloading the DB (which isn't the default behavior).